curve25519-dalek/Cargo.toml

```diff
@@ -83,6 +83,7 @@ legacy_compatibility = []
 group = ["dep:group", "rand_core"]
 group-bits = ["group", "ff/bits"]
 digest = ["dep:digest"]
+lizard = ["digest"]
```

Comment:
Nitpick: `lizard` doesn't have to depend on `digest`, since the plain `map_to_curve[_inverse]` work on bytestrings. (Arguably, those operations are merely "Elligator 2" and not "Lizard" anyway.)